.github/workflows/os_comp.yml

```diff
@@ -130,6 +133,15 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: Get zig
+        if: ${{ contains(matrix.cfg.CC, 'zig') }}
+        run: |
+          ZIGVER=${{ matrix.cfg.VERSION }}
+          wget https://ziglang.org/download/$ZIGVER/zig-$ZIGVER.tar.xz
+          tar xf "zig-$ZIGVER.tar.xz"
+          cd "zig-$ZIGVER"
+          echo "PATH=$PWD:$PATH" >> $GITHUB_ENV
```

Comment:
Why not do this in ci/ciimage/ubuntu-rolling/install.sh ?

Reply:
Mostly done. Should zig go right into the tools dir or should a path be added to the directory to where it's extracted?

Reply:
```
# As mentioned in the Zig readme, the binary and files under lib can be copied
# https://github.com/ziglang/zig?tab=readme-ov-file#installation
mv zig /usr/bin
mv -a lib /usr/lib/zig
```